x-pack/test/ui_capabilities/security_and_spaces/tests/saved_objects_management.ts

```diff
@@ -13,7 +12,7 @@ import {
 } from '../../common/services/ui_capabilities';
 import { UserAtSpaceScenarios } from '../scenarios';
 
-const savedObjectsManagementBuilder = new SavedObjectsManagementBuilder(true);
+const savedObjectsManagementBuilder = new SavedObjectsManagementBuilder();
 
 // eslint-disable-next-line import/no-default-export
 export default function savedObjectsManagementTests({
```

Comment:
I'm tempted to say we should delete these tests entirely, as "savedObjectsManagement" just ends up becoming a normal feature. What do you think?

Reply:
Yep, I thought about deleting them too. I left them in so that I wasn't asked to put them back, but it's easy enough to take out. Three less suites to manage going forward!

Reply:
Let's get rid of them then, I don't think they get us much.